.github/workflows/integration.yml

```diff
@@ -2,14 +2,14 @@ name: Integration Tests
 
 on:
   pull_request:
-    types: [synchronize, opened, reopened, ready_for_review]
+    types: [synchronize, opened, reopened, ready_for_review, labeled]
```

Comment:
I think this should only be triggered with `labeled` or `synchronize`.

Reply:
I have no opinion on this because the integration test will only trigger with the label attached. If someone changes the state of the PR then this gets re-evaluated.

Reply:
It makes sense to me to remove the `opened, reopened, ready_for_review` types and only leave `synchronize, labeled`.  AFAIK this way, if a PR is opened without the label, there will be no entries in the list of checks related to the jobs in this workflow, which looks cleaner to me.